src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -518,7 +518,7 @@ static bool tile_is_inside_clip_view(const tile_descriptor& tile)
     if (tile.tile_element == nullptr)
         return false;
 
-    if (tile.tile_element->GetBaseZ() > gClipHeight / COORDS_Z_STEP)
+    if (tile.tile_element->GetBaseZ() > gClipHeight * COORDS_Z_STEP)
```

Comment:
I think if you revert the assignment in ViewClipping and just remove the code `/ COORDS_Z_STEP`, we have the same result. Not tested yet.

Reply:
Absolutely not. gClipHeight contains small Z coordinates, except for the one or two assignments and comparisons I mistakingly changed in earlier PRs.